modules/oadp-gcp-wif-cloud-authentication.adoc

```diff
@@ -17,10 +17,9 @@ Workload identity federation handles encrypting and decrypting certificates, ext
 Google workload identity federation is available for OADP 1.3.x and later.
 ====
 
-[NOTE]
-====
-For backing up volumes, OADP on GCP with Google workload identity federation authentication supports only CSI snapshots.
-====
+For backing up volumes, OADP on GCP with Google workload identity federation authentication only supports CSI snapshots.
```

Comment:
I know this is existing text, should this be _When_? Currently, it sounds like the sentence refers to _volumes_ rather than _backing up_.

```suggestion
When backing up volumes, OADP on GCP with Google workload identity federation authentication only supports CSI snapshots.
```